server-phoenix/mix.exs

```diff
@@ -54,7 +54,7 @@ defmodule Helios.MixProject do
       {:dotenv, "~> 3.0.0", only: [:dev, :test]},
       {:tzdata, "~> 1.1"},
       {:httpoison, "~> 1.7"},
-      {:sun_times, "~> 0.1.2"},
+      {:sun_times, git: "https://github.com/mojotech/sun_times.ex"},
```

Comment:
Fine with us linking to the mojotech repos for now. Can you make a PR for the changes to upstream?

Reply:
Done!! I'll update our reference back to the original if my PR is ever approved.